### DISCUSSION TITLE
New to GitHub - Line Breaks on READMEs?

### DISCUSSION THREAD

**Discussion by CPang23** [community/community#18966] [Other Feature Feedback, Questions, & Ideas] [answered]
I just created my GitHub account and I'm in the middle of making my README for my profile. Maybe I'm just clueless, but how do you do a line break for formatting purposes? Can't find the answer online...

**Comment by justin-2028** [comment, marked as answer]
Just add< br / > (without the spaces between the angle brackets) to the end of a sentence for a line break.

This link may help with further elaboration:
https://stackoverflow.com/questions/24575680/new-lines-inside-paragraph-in-readme-md

**Reply by CPang23** [reply]
Thanks, you a real one 💪

**Reply by justin-2028** [reply]
np

**Comment by thuongtruong1009** [comment]
Hi @CPang23 
Simply that you using <br/> tag

**Comment by itzmzn** [comment]
mzn

**Comment by itzmzn** [comment]
mznking

**Comment by Willtech** [comment]
With Markdown the correct way to insert a line break - even though it supports HTML in the usual rweaders- is to type two spaces  
and then press return.

Professor. Damian A. James Williamson

**Comment by yasasbanukaofficial** [comment]
Sometimes the **br** doesn't work so use **&nbsp ;**   (remove the space between the nbsp and the semicolon) in a line where you want to use. Works perfectly.
